MREUnityRuntime/MREUnityRuntimeLib/Constants.cs

```diff
@@ -60,6 +60,6 @@ static Constants()
 		 * Work in progress feature to improve shared physics synchronization.
 		 * It is NOT compatibile with SDK versions older than 0.19.
 		 */
-		internal const bool UsePhysicsBridge = false;
+		internal const bool UsePhysicsBridge = true;
```

Comment:
switch to true?

Reply:
I would say that is up to you all for when you feel this is ready to enable for all users.  If we are ready, we should probably just remove this and the checks altogether, right?

---
In reply to: [449570505](https://github.com/microsoft/mixed-reality-extension-unity/pull/210#discussion_r449570505) [](ancestors = 449570505)

Reply:
Agree, let's keep it for few weeks and then remove it

---
In reply to: [450334859](https://github.com/microsoft/mixed-reality-extension-unity/pull/210#discussion_r450334859) [](ancestors = 450334859,449570505)